ffi/capi/src/neo_datetime.rs

```diff
@@ -506,6 +479,292 @@ pub mod ffi {
             )))
         }
 
+        #[diplomat::rust_link(icu::datetime::fieldsets::zone::GenericShort, Struct)]
+        #[cfg(feature = "compiled_data")]
+        pub fn with_zone_generic_short(
```

Comment:
`with_zone_generic`? I remember we had these names on the Rust API at some point, and ECMA also uses 'generic' and 'genericLong'

Reply:
https://github.com/unicode-org/icu4x/issues/5940#issuecomment-2722698945